scripts/deployUpgradeTest.ts

```diff
@@ -0,0 +1,51 @@
+import hre, { ethers } from "hardhat";
+import { BigNumber } from "ethers";
+import { expect } from "chai";
+import { stopImpersonate, impersonateAccount, fundAccount } from "../integrationTests/IntegrationTestContext";
+
+const About4700ETH = "0x1000000000000000000";
```

Comment:
this value will change with price of ETH? is there a way to make it more human readable and constant?